.github/workflows/deploy-daac.yml

```diff
@@ -31,6 +31,7 @@ jobs:
               job_spec/INSAR_ISCE_BURST.yml
               job_spec/INSAR_ISCE_MULTI_BURST.yml
               job_spec/ARIA_S1_GUNW.yml
+              job_spec/OPERA_DISP_TMS.yml
```

Comment:
Will this job type still fail at the files step? Seems weird for it to be publicly available if it's gonna fail.

Reply:
Yes and yes, but I don't see a recourse.